roles/leadership/event-coordinator.md

```diff
@@ -1,19 +1,22 @@
-# The Black Python Devs Website Manager
+# Events Coordinator
 
-## Current Role Leader: Velda Kiara
-The Black Python Devs Events Coordinator for various activities of the community 
+The Events Coordinator is responsible for managing various community activities for Black Python Devs.
 
-## Responsibilities 
-* Ensure all event logistics, such as scheduling, communication, and technical requirements, are well-organized.
+## Current Role Leader
+
+* [Velda Kiara](https://github.com/VeldaKiara)
```

Comment:
@activus-d - This wasn't announced but as of PyCon US @VeldaKiara  will no longer be the events coordinator. We will begin looking for a new person to coordinate events.

She will continue to serve as the vice-chair of BPD.